lib/kubeclient/common.rb

```diff
@@ -152,15 +151,15 @@ def watch_entities(entity_type, resource_version = nil)
       Kubeclient::Common::WatchStream.new(uri, options)
     end
 
-    def get_entities(entity_type, klass, options)
+    def get_entities(entity_type, klass, namespace = nil, options = {})
```

Comment:
Maybe we could just include namespace in the options hash. We could also do the same for the specific implementations of `get_#{entity_type}`. I'm torn on this though, namespace is specified fairly often, so it's nice that it's treated separately. But, it's also possible for it to be nil, in which case it might be nice to not have to pass a nil parameter around.